aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -109,9 +109,10 @@ class DeleteModal extends Component {
                 }}
               />
               <label htmlFor='check1'>
-                <FormattedHTMLMessage
+                <FormattedMessage
                   id='modal.delete.submissions.text'
-                  defaultMessage='Data <b>submitted to</b> this pipeline (Submissions)'
+                  defaultMessage='Data <b>submitted to</b> this pipeline (Submissions)'
```

Comment:
Do not remove the whitespace I :wink:

Reply:
What happens to your whitespaces? :thinking:

Reply:
The trailing whitespaces gets trimmed after using rich text formatting.. without this workaround, the text reads `Data*submitted to*this pipeline ...` instead of `Data *submitted to* this pipeline ...`

Reply:
:flushed:

Reply:
[Checkout this comment] (https://github.com/formatjs/babel-plugin-react-intl/issues/77#issuecomment-250254247)

Reply:
What I meant is that it looks like another type of whitespace... do you see how they have different length?

![image](https://user-images.githubusercontent.com/8459537/66758954-98116b80-ee9f-11e9-9f7b-9af6085a2a81.png)

Reply:
Yeah.. [I copied it from here](https://beautifuldingbats.com/whitespace)
If it's ok using it.. i'll then use another that match up with our initial space

Reply:
:thinking: 

I would prefer the whitespace that appear pressing the keyboard space bar... but obviously if you prefer this more, keep it. :man_artist: